runtime/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -19,7 +19,7 @@ import kotlin.reflect.*
  * or for converting objects to [JsonElement] back and forth.
  *
  * This is the only serial format which has first-class [JsonElement] support.
- * Any serializable class can be serialized to or from [JsonElement] with [Json.fromJson] and [Json.toJson] respectively or
+ * Any serializable class can be serialized to or from [JsonElement] with [Json.decodeFromJsonElement] and [Json.encodeToJsonElement] respectively or
```

Comment:
The current docs _don't mention that this was changed_ in 1.7.3: https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/decode-from-json-element.html

And there appears to be **no way to see docs for older versions of the library** from the web-based docs viewer: https://discuss.kotlinlang.org/t/language-docs-for-previous-kotlin-versions/28800 

… which combines to make the Kotlin developer experience very painful for those of us who can't use the latest-version-of-everything.

Python really gets these documentation and versioning issues right in a way that Kotlin doesn't: https://discuss.kotlinlang.org/t/language-docs-for-previous-kotlin-versions/28800/2

Reply:
This is a 2020 PR with pre-1.0 names. Can you please clarify what exactly was changed in 1.7.3?

Reply:
Apologies, I completely misread the tags and the _name_ change apparently occurred far earlier 🤦🏻